### PR TITLE
Removed thematic areas ui

### PR DESCRIPTION
**Pull request by nerik**
Part of  https://github.com/NASA-IMPACT/veda-ui/issues/494

### CODE REVIEW

**Comment by danielfdsilva** [comment]
@nerik Added some work on top of what you had started. Nothing changes the current behavior despite the large amount of changes (module renaming).
I tried to split the changes as best I could into individual commits.

File: app/scripts/components/analysis/results/chart-card.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ import {
 } from '$components/common/chart/analysis/utils';
 import DropMenuItemButton from '$styles/drop-menu-item-button';
 import { getDatasetPath } from '$utils/routes';
-import { ThematicItemFull, useThematicArea } from '$utils/thematics';
+import {  allDatasetsProps } from '$utils/veda-data';
```
Comment (nerik) [inline app/scripts/components/analysis/results/chart-card.tsx:46]: (nit)
```suggestion
import { allDatasetsProps } from '$utils/veda-data';
```

**Review by nerik — COMMENTED** [review]
LGTM 🚀

**Review by danielfdsilva — APPROVED** [review]

**Comment by danielfdsilva** [comment]
@karitotp With this PR thematic areas are no longer part of the VEDA dashboard.
Datasets and Discoveries exist globally without being grouped. Everything else is the same. Can you do some quick QA checks to see everything still behaves correctly?

**Pull request merged**